common/djangoapps/status/models.py

```diff
@@ -26,7 +26,7 @@ class GlobalStatusMessage(ConfigurationModel):
     message = models.TextField(
         blank=True,
         null=True,
-        help_text='<p>The contents of this field will be displayed as a warning banner on all views.</p>'
+        help_text=u'<p>The contents of this field will be displayed as a warning banner on all views.</p>'
                   '<p>To override the banner message for a specific course, refer to the Course Message configuration. '
```

Comment:
These strings are concatenated together by Python; probably best to explicitly make them all `u`-prefixed for clarity?